bootstrap/command/pr

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 The Absolute Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+# Check for token & current branch.
+TOKEN=$(git config --local --get PAT.Token)
+USER_ID=$(git config --local --get PAT.User)
+BR_O=$(git symbolic-ref --short HEAD)
+
+# Upload 'pull-request' on the command-line.
+# Before running this, a new commit must be pushed to your origin repo.
+if [ $1 = 'upload' ]
+then
+	if [ -z $TOKEN ]
+	then
+		echo "Only the first attempt require your ID and password."
+		read ID
+		read PW
+		GITHUB_TOKEN=$(curl -X POST -u "$ID:$PW" -d '{"note":"Token","scopes":["repo"]}' "https://api.github.com/authorizations" | grep -Pom 1 '"token": "\K[^"]*')
+
+		if [ -z $GITHUB_TOKEN ]
+		then
+			echo "Connection Error"
+		else
+			git config --local --add PAT.User $ID
+			git config --local --add PAT.Token $GITHUB_TOKEN
+			echo "Token is created. Please run command './absilute pr upload or #num' again."
+		fi
+	else
+		echo "Please write the title and contents of the pull-request."
+		read TITLE
+		read BODY
+		echo "Enter upstream's branch name"
+		read BR_U
+
+		touch curl_data.json
+		echo "{\"title\" : \"$TITLE\",\"body\" : \"$BODY\",
+		  \"head\" : \"$USER_ID:$BR_O\",\"base\" : \"$BR_U\"}" >> curl_data.json
```

Comment:
The $USER_ID:$BR_O will be only supported for id based branch.
I mean that some members can make feature branches in this repository directly.
So, if you would be a member of this project, you can make a romandev:hello branch instead of rockplani:hello branch. And then you can compare two branches, romandev:hello and romandev:master. (and then make a pull request)